google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/ManagedCloudSdkService.java

```diff
@@ -55,12 +52,13 @@
 
   private final Collection<SdkStatusUpdateListener> statusUpdateListeners = Lists.newArrayList();
 
-  private volatile ListenableFuture<Path> runningInstallationJob;
+  private volatile ListenableFuture<Path> managedSdkBackgroundJob;
 
   @Override
   public void activate() {
     // TODO track event that custom SDK is activated and used.
 
+    initManagedSdk();
     install();
```

Comment:
I may have asked the same thing in the last PR (sorry); but why does activate call `install`? What if its already installed?

Reply:
`install()` checks first if it's already installed and then checks if App Engine Component is already installed. It may sound a bit ambigious, maybe introduce `checkInstallStatus()` method or something  like that, or bake that into `initManagedSdk()`?

Reply:
is the idea that it will also update it if its out of date (and auto-update is enabled)?

I think extracting it out into the init method makes sense. It will make it more explicit and easier to trace whats going on.

Reply:
Yes, I think we planned to do update as well when plugin starts or managed SDK is activated first time. I think it's a good idea to add a new method.